template-for-repository/proofs/Makefile.common

```diff
@@ -339,6 +344,18 @@ $(HARNESS_GOTO).goto: $(HARNESS_GOTO)4.goto
 goto: $(HARNESS_GOTO).goto
 	echo $(DEPENDENT_GOTOS)
 
+################################################################
+# Targets to run Arpa
+
+arpa_cmake_project:
```

Comment:
I think this target should actually be called `$(ARPA_BLDDIR)`, since that is what will be created

Reply:
I guess you want the project owner to set `ARPA_CMAKE_FLAGS` to be the same as the cmake flags used for building the project?

If so, please add a comment to Makefile-project-defines to tell people to set this variable, similar to the `ARPA` comment I mentioned above